src/map/clif.c

```diff
@@ -1155,10 +1155,20 @@ static void clif_set_unit_idle(struct block_list *bl, struct map_session_data *t
 	p.font = (sd) ? sd->status.font : 0;
 #endif
 #if PACKETVER >= 20120221
-	if (battle_config.show_monster_hp_bar && bl->type == BL_MOB && status_get_hp(bl) < status_get_max_hp(bl)) {
+	const struct mob_data *md = BL_UCCAST(BL_MOB, bl);
```

Comment:
``md`` already exists below in clif.c:1176. you cant define same variable name more than once in one code block